plugins/modules/efs.py

```diff
@@ -767,6 +818,9 @@ def main():
                                                   throughput_mode=throughput_mode, provisioned_throughput_in_mibps=provisioned_throughput_in_mibps) or changed
         if transition_to_ia:
             changed |= connection.update_lifecycle_policy(name, transition_to_ia)
+        if backup:
+            backup = str(backup).upper()
+            changed = connection.change_backup_policy(name, backup)
```

Comment:
It should `changed |=`, not just `=`, this way we preserve the value if it's set `True` above.

e.g:
```python
>>> changed = False
>>> changed |= False
>>> changed |= True
>>> changed |= False
>>> changed
True
```